src/main/java/uk/gov/hmcts/reform/cpo/errorhandling/AuthError.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.cpo.errorhandling;
+
+/**
+ * Authentication and Authorisation error messages.
+ **/
+public final class AuthError {
+
+    public static final String AUTHENTICATION_TOKEN_INVALID
+        = "Authentication failure due to invalid / expired tokens (IDAM / S2S).";
+
+    public static final String UNAUTHORISED_S2S_SERVICE = "Unauthorised S2S service.";
+
+    // Hide Utility Class Constructor : Utility classes should not have a public or default constructor (squid:S1118)
+    private AuthError() {
```

Comment:
I am not sure but  lombok could gives us some facilities.    for final classes with private constructors 
@Value(staticConstructor = "getInstance")  However this is an small comments not mandatory.

Reply:
Class copied from MCA, but will investigate

Reply:
Matt has resolved it :) .  It will be fixed on master